### PR TITLE
adding designer url fix

### PR DESCRIPTION
**Pull request by nuwan-samarasinghe**
adding designer url fix

### CODE REVIEW

File: copilot/fsd-form-designer-adapter/manifest.yml [hunk 1]
```diff
@@ -51,23 +51,10 @@ network:
...
-  LOGOUT_URL: "https://authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk/sessions/sign-out"
-  MULTIFUND_URL: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk/account"
-  NODE_CONFIG: '{"safelist": ["fsd-application-store"]}'
   NODE_ENV: production
-  PRIVACY_POLICY_URL: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk/privacy"
-  SERVICE_START_PAGE: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk/account"
-  SINGLE_REDIS: true
-  JWT_AUTH_COOKIE_NAME: "fsd_user_token"
-  ELIGIBILITY_RESULT_URL: "https://frontend.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk/eligibility-result"
+  CHOKIDAR_USEPOLLING: true
```
Comment (sfount) [inline copilot/fsd-form-designer-adapter/manifest.yml:55]: Will this be used in the production environment? I've mostly seen this as a method for polling the filesystem to restart the server on changes but just want to sense check if thats something we want in the deployment environments

**Review by sfount — COMMENTED** [review]
Change looks all good! Would it be possible to add a commit message briefly explaining why these variables aren't actually relevant to this manifest?

File: copilot/fsd-form-designer-adapter/manifest.yml (hunk 1 above)
Comment (nuwan-samarasinghe) [reply copilot/fsd-form-designer-adapter/manifest.yml:55]: designer any way will not be in prod environment but in in the prod section of the yml we override this

**Review by sfount — APPROVED** [review]
Approving to keep things moving, from what I understand the environment settings here were moved over from the runner to the designer but aren't actually consumed by the designer. This removes them to keep only relevant flags.

(If the commit message is updated happy to reapprove!)

**Pull request merged**